src/style.css

```diff
@@ -0,0 +1,136 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.search {
+  margin: 20px 0;
+  position: relative;
+}
+
+.search_big {
+  font-family: "Avenir", sans-serif;
+  display: block;
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-size: 16px;
+  font-weight: 300;
+  padding: 25px 709px 23px 62px;
+  color: #3d4e61;
+  line-height: 22px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search_big:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+}
+
+.search_big:focus {

```

Comment:
don't repeat a lot of times the same properties for different state
 in `focus` must be only different properties from main state